### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Thundermaker300**

### CODE REVIEW

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 1]
```diff
@@ -167,7 +167,7 @@ public static void SetRoomColorForTargetOnly(this Room room, Player target, Colo
         }
 
         /// <summary>
-        /// Set <see cref="FlickerableLightController.Network_lightIntensityMultiplier"/> that only <see cref="Player"/> can see.
+        /// Set's <see cref="Room.Color"/> of a <paramref name="room"/> that only the <paramref name="target"/> player can see.
```
Comment (Ficus-x) [inline Exiled.API/Extensions/MirrorExtensions.cs:170]: Set`s 0_o?

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 2]
```diff
@@ -155,7 +155,7 @@ public static void PlayGunSound(this Player player, Vector3 position, ItemType i
         }
 
         /// <summary>
-        /// Set <see cref="FlickerableLightController.Network_warheadLightColor"/> that only <see cref="Player"/> can see.
+        /// Set's <see cref="Room.Color"/> of a <paramref name="room"/> that only the <paramref name="target"/> player can see.
```
Comment (Jesus-QC) [inline Exiled.API/Extensions/MirrorExtensions.cs:158]: Sets

File: Exiled.API/Extensions/MirrorExtensions.cs (hunk 1 above)
Comment (Jesus-QC) [inline Exiled.API/Extensions/MirrorExtensions.cs:170]: Sets

**Review by Jesus-QC — CHANGES_REQUESTED** [review]

File: Exiled.API/Extensions/MirrorExtensions.cs (hunk 1 above)
Comment (Thundermaker300) [reply Exiled.API/Extensions/MirrorExtensions.cs:170]: tired brain moment, will fix tomorrow probably

**Review by rus-skill-issue — APPROVED** [review]

**Comment by PintTheDragon** [comment]
Imo "or not" is redundant, but it's your choice if you want to include it.

**Comment by Thundermaker300** [comment]
> Imo "or not" is redundant, but it's your choice if you want to include it.

Some docs have it, some have "whether [property info here] or not", some don't have either. I'm just trying to make it consistent lol

**Review by PintTheDragon — APPROVED** [review]

**Comment by Thundermaker300** [comment]
Moved to Exiled EA

**Pull request closed without merging**